tensorflow/core/api_def/base_api/api_def_SamplingDataset.pbtxt

```diff
@@ -22,8 +22,10 @@ END
   }
   summary: "Creates a dataset that takes a Bernoulli sample of the contents of another dataset."
   description: <<END
-There is no direct Python API for creating Datasets of this type. Rewrite rules in 
-`tensorflow/core/grappler/optimizers/data/filter_with_random_uniform_fusion.cc`
-create instances of this dataset.
+There is no direct Python API for creating Datasets of this type.  
```

Comment:
"There is no transformation in the tf.data Python API for creating this dataset. Instead, it is created as a result of the` filter_with_random_uniform_fusion` static optimization. Whether this optimization is performed is determined by the `experimental_optimization.filter_with_random_uniform_fusion` option of `tf.data.Options`."

Reply:
Thanks for the corrections. I've pushed a commit that updates the docs accordingly.